test/parallel/test-fork-send-returns-boolean.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+
+const n = fork(common.fixturesDir + '/empty.js');
+
+const rv = n.send({ hello: 'world' });
+assert.strictEqual(rv, true);
```

Comment:
It's forking an empty file. The child has no listeners etc. so it exits immediately.